integration/common/openlineage/common/provider/dbt/structured_logs.py

```diff
@@ -452,10 +457,18 @@ def _setup_dbt_run_metadata(self, start_event):
         The dbt command defines the parent run metadata of
         all subsequent OL events (NodeStart, NodeFinish ...)
         """
+        root_parent_run_id = start_event.run.runId
+        if start_event.run.facets.get("parent"):
+            parent_facet = start_event.run.facets.get("parent")
+            if hasattr(parent_facet, "root"):
+                root_parent_run_id = parent_facet.rootParentRunId
```

Comment:
Sorry, cannot find this facet anywhere in current repo. Could you please add a comment about this case?

Reply:
Fixed